components/auth/magic-word-form.tsx

```
@@ -0,0 +1,213 @@
+        animate={{ opacity: 1, y: 0 }}
+        transition={{ delay: 0.3 }}
+        className="p-6 bg-gradient-to-br from-gray-900/80 to-gray-800/50 rounded-2xl border border-purple-500/30 shadow-xl shadow-purple-500/10"
+      >
+        <div className="space-y-5">
+          {/* Magic Word Input */}
+          <div>
+            <label className="block text-sm font-medium text-purple-300 mb-2">
+              <Sparkles className="inline w-4 h-4 mr-1" />
+              Magic Word
+            </label>
+            <div className="relative">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)